baklava/src/utils/keybinds/worker.ts

```diff
@@ -2,14 +2,11 @@ import { isMainThread, parentPort } from 'worker_threads';
 import globkey from 'globkey';
 
 if (!isMainThread) {
-    let prev_keys = ['', '']
     let shouldBreak = false;
+    globkey.start();
     while (true) {
         if (shouldBreak) break;
         let keys = globkey.getKeys();
-        if (keys != prev_keys) {
-            if (parentPort) parentPort.postMessage({ type: 'keys', keys: keys });
-        }
-        prev_keys = keys
+        if (parentPort) parentPort.postMessage({ type: 'keys', keys: keys });
     }
-}
\ No newline at end of file
+}
```

Comment:
didn't you say you created a worker in rust? do we still have to use the worker i made in js and spent 7 hours to get working on? *(please say yes, otherwise i will die)*

Reply:
nvm i tested this branch out locally and it works.